src/formatDate.js

```diff
@@ -8,7 +8,36 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let resultDate = '';
+  const objectDateFormat = {};
+  const dateSplitted = date.split(fromFormat[fromFormat.length - 1]);
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    const less = dateSplitted[i] < 30;
+    const more = dateSplitted[i] >= 30;
+
+    if (fromFormat[i] === 'YY' && toFormat.includes('YYYY') && less) {
+      objectDateFormat.YYYY = '20' + dateSplitted[i];
+    } else if (fromFormat[i] === 'YY' && toFormat.includes('YYYY') && more) {
+      objectDateFormat.YYYY = '19' + dateSplitted[i];
+    }
+
+    if (fromFormat[i] === 'YYYY' && toFormat.includes('YY')) {
+      objectDateFormat.YY = dateSplitted[i][2] + dateSplitted[i][3];
```

Comment:
The conversion from 'YYYY' to 'YY' is correct, but you should ensure that 'dateSplitted[i]' is a string and has a length of 4 before accessing its characters to avoid potential errors.